library/evaluation/calibration.py

```diff
@@ -35,7 +35,9 @@ def __calibration_plot(y_true: List, predicted: List[PredsWithProbs]):
     return fig
 
 
-calibration_metrics: Evaluators = [
-    ("calibration", __calibration_report),
-    ("calibration_plot", __calibration_plot),
-]
+# calibration_metrics: Evaluators = [
```

Comment:
hmm I do rely on these. can we create a function for it, that returns them?

Reply:
I will put that back in, I just commented it out on my example!